install.sh

```diff
@@ -172,15 +172,19 @@ install_nvm_as_script() {
   nvm_download -s "$NVM_SOURCE_LOCAL" -o "$INSTALL_DIR/nvm.sh" || {
     echo >&2 "Failed to download '$NVM_SOURCE_LOCAL'"
     return 1
-  }
+  } &
   nvm_download -s "$NVM_EXEC_SOURCE" -o "$INSTALL_DIR/nvm-exec" || {
     echo >&2 "Failed to download '$NVM_EXEC_SOURCE'"
     return 2
-  }
+  } &
   chmod a+x "$INSTALL_DIR/nvm-exec" || {
     echo >&2 "Failed to mark '$INSTALL_DIR/nvm-exec' as executable"
     return 3
-  }
+  } &
+  for job in $(jobs -p | sort)
+  do
+    wait "$job" || return $?
```

Comment:
I've not had good luck with `wait` reliably waiting on all parallel commands :-/